spot-client/src/common/remote-control/BaseRemoteControlService.js

```diff
@@ -181,7 +181,7 @@ export class BaseRemoteControlService extends Emitter {
             return `${prefix}-${generate8Characters()}`;
         }
 
-        return undefined;
+        return `remote-${generate8Characters()}`;
```

Comment:
how about doing `remote-perm` only on permanent and do that fallback in any other case including temporary remote ?